src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,26 @@
-export const TodoFilter = () => (
+type Props = {
+  selectedFilter: string,
+  setSelectedFilter: (select: string) => void,
```

Comment:
apply for similar cases as well
```suggestion
  setSelectedFilter: React.Dispatch<React.SetStateAction<string>>,
```
or you can destructure React object

Reply:
done